src/main/java/Event.java

```diff
@@ -0,0 +1,25 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task{
+    protected LocalDate from;
+    static DateTimeFormatter format = DateTimeFormatter.ofPattern("MMM d yyyy");
+    protected LocalDate to;
```

Comment:
you may want to consider rearranging the order, perhaps static variables together and non static ones together.